tpm/tpm.go

```diff
@@ -34,27 +32,9 @@ type Closer interface {
 // extensive resource management that the kernel already handles for us
 // when using the kernel resource manager.
 func Open(path string) (Closer, error) {
```

Comment:
I think we should call this `OpenUnix` to make it clear that TCP and Simulator openers should be separate funcs and not combined into the functionality of `Open`.

Reply:
I split the difference on the different direction. I did remove Open entirely to just depend on the go-tpm transport interface, but it felt weird to have their interface as a param to go-fdo's functions, so I aliased it.